pubspec.yaml

```diff
@@ -99,7 +99,7 @@ dependencies:
   device_info: 2.0.3
 
   # permission_handler
-  permission_handler: 8.1.6
+  permission_handler: 8.2.6
```

Comment:
same here, we need to understand what is update with new version and carefully test in many OS versions